Graphics/Sudbury/WirePackages.hs

```diff
@@ -26,37 +24,33 @@ data WirePackage = WirePackage
   , wirePackagePayload :: B.ByteString
   } deriving (Eq, Show)
 
--- | Construct a wayland wire wirePackage
-wirePack :: WirePackage -> BB.Builder
-wirePack msg = wirePackBuilder
-  (wirePackageSender msg)
-  (wirePackageSize msg)
-  (wirePackageOpcode msg)
-  (BB.byteString $ wirePackagePayload msg)
-
--- | Construct a wayland wire wirePackage with payload given as a 'Builder'.
-wirePackBuilder :: Word32 -> Word16 -> Word16 -> BB.Builder -> BB.Builder
-wirePackBuilder sender size opcode payload =
-  BBE.word32Host sender
-  <>
-  -- FIXME make byte order portable here
-  BBE.word16Host opcode
-  <>
-  BBE.word16Host size
-  <>
-  payload
+-- Storable that writes in the Wayland binary format.
+-- Allows us to use the optimized machinery for Data.Store
+-- to gain faster performing decoding/encoding.
```

Comment:
This is a useful comment, so make it a haddock one by starting with `-- |`. "Faster performing" than what?